wai-aria/role/invalid-roles.html

```diff
@@ -8,29 +8,127 @@
   <script src="/resources/testdriver-vendor.js"></script>
   <script src="/resources/testdriver-actions.js"></script>
   <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <meta charset="utf-8">
 </head>
 <body>
 
-<p>Verifies Invalid Roles from <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a></p>
+<p>Tests invalid ARIA roles from <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a></p>
```

Comment:
M. Seeing if an irrelevant change will kick the build checks off properly.
```suggestion
<p>Tests invalid ARIA roles from <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a>.</p>
```